imagebitmap-renderingcontext/detached-source.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
could you put transferFromImageBitmap in the file name?
Something like transferFromImageBitmap-detached.html